app/views/grade_entry_forms/_react_grades_table.js.jsx.erb

```diff
@@ -40,9 +40,12 @@
     },
     render: function() {
       return(
-        <input id={this.props.grade_id} type="text" size="4" 
-         className="grade-input" value={this.props.value} 
-         onClick={this.handleChange} readonly/>
+        <p 
```

Comment:
@wkwan Can you change this to a `span`? The `p` has vertical padding that I don't like the look of.